processing/src/main/java/org/apache/druid/jackson/DruidDefaultSerializersModule.java

```diff
@@ -189,20 +188,7 @@ public ByteOrder deserialize(JsonParser jp, DeserializationContext ctxt) throws
     );
     addDeserializer(ResponseContext.class, new ResponseContextDeserializer());
 
-    addSerializer(RowsAndColumns.class, new JsonSerializer<RowsAndColumns>()
-    {
-      @Override
-      public void serialize(
-          RowsAndColumns value,
-          JsonGenerator gen,
-          SerializerProvider serializers
-      ) throws IOException
-      {
-        // It would be really cool if jackson offered an output stream that would allow us to push bytes
-        // through, but it doesn't right now, so we have to build a byte[] instead.  Maybe something to contribute
-        // back to Jackson at some point.
-        gen.writeBinary(WireTransferable.fromRAC(value).bytesToTransfer());
```

Comment:
can we retire `WireTransferable` ?
with these changes it will be only used in [WindowOperatorQueryQueryRunnerFactory](https://github.com/apache/druid/blob/40f38f0191dcd03b755224c2fb604ad732c7a8c7/processing/src/main/java/org/apache/druid/query/operator/WindowOperatorQueryQueryRunnerFactory.java#L107-L115) 
but the usecase there seem to closely match `rac.as(FrameRowsAndColumns.class)` or not?

Reply:
Removed `WireTransferable`